spec/features/user_spec.rb

```diff
@@ -12,35 +13,45 @@
         expect(page).to have_content(user.name)
       end
 
-      it 'allows create cards' do
-        create_card
+      it 'allows to create cards in deck' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.